phpseclib/Net/SSH2.php

```diff
@@ -652,7 +652,7 @@ class SSH2
      * @see self::_get_channel_packet()
      * @access private
      */
-    private $curTimeout;
+    protected $curTimeout;
```

Comment:
Please minimize these changes to those that are required, since the phpdoc changes are bug fixes. :)